docs/en/01_basic-configuration.md

```diff
@@ -34,26 +39,29 @@ SilverStripe\Core\Injector\Injector:
 ```
 
 ## Excluding page types
-By default, all pages which inherit from `SiteTree` will be included in the static publishing system.  This can cause issues on some pages, especially those which contain a CSRF token (eg. forms) since the token will also be cached and hence prevent submissions from being validated.
 
-You can exclude pages from being statically generated on a class-by-class basis by adding a `urlsToCache()` method to your page class which returns an empty array:
+By default, all pages which inherit from `SiteTree` will be included in the static publishing system. This can cause issues on some pages, especially those which contain a CSRF token (eg. forms) since the token will also be cached and hence prevent submissions from being validated.
+
+You can exclude pages from being statically generated on a class-by-class basis by adding a [`StaticallyPublishable::urlsToCache()`](api:SilverStripe\StaticPublishQueue\Contract\StaticallyPublishable::urlsToCache()) method to your page class which returns an empty array:
 
 ```php
+namespace App\Pages;
+
 class MyFormPage extends Page
```

Comment:
```suggestion

use Page

class MyFormPage extends Page
```
I won't add a comment for every code block - please make sure they all get the correct `use` statements added.

Reply:
Not changing, php docs linter will complain about the lack of a namespace

Reply:
Updated